documentation.html

```diff
@@ -0,0 +1,828 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="component.css">
+    <link rel="stylesheet" href="documentation.css">
+    <link rel="shortcut icon" href="./Images/icon.png" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Documentation</title>
+</head>
+
+<body>
+
+    <!-- Hamburger -->
+    <div class="hamburgerHolder">
+        <img class="logo" src="./Images/icon.png" alt="">
+        <div class="burgerbox">
+            <div class="burgerlines firstcross"></div>
+            <div class="burgerlines middledash"></div>
+            <div class="burgerlines lastcross"></div>
+        </div>
+    </div>
+    <div class="container">
+        <section class="content-section">
+            <h1>Contents</h1>
+            <ul>
+                <li class="contents"><a href="#avatar">Avatar</a></li>
+                <li class="contents"><a href="#alert">Alert</a></li>
+                <li class="contents"><a href="#badge">Badge</a></li>
+                <li class="contents"><a href="#buttons">Buttons</a></li>
+                <li class="contents"><a href="#cards">Cards</a></li>
+                <li class="contents"><a href="#image">Image</a></li>
+                <li class="contents"><a href="#list">Lists</a></li>
+                <li class="contents"><a href="#input">Input</a></li>
+                <li class="contents"><a href="#modal">Modal</a></li>
+                <li class="contents"><a href="#nav">Navigation</a></li>
+                <li class="contents"><a href="#rating">Rating</a></li>
+                <li class="contents"><a href="#textutil">Text Utilities</a></li>
+                <li class="contents"><a href="#grid">Simplified grid</a></li>
+                <li class="contents"><a href="#toast">Toast/Snackbar</a></li>
+            </ul>
+        </section>
+        <section class="documentation">
```

Comment:
Align and size your website probably. A genral rule you can apply is to have evrything aligned to one basis line.